man/org.freedesktop.resolve1.xml

```diff
@@ -115,6 +115,7 @@ node /org/freedesktop/resolve1 {
       ResetStatistics();
       FlushCaches();
       ResetServerFeatures();
+      ListDiscovered(out a(stiiay) UNNAMED);
```

Comment:
Shouldn't the method name somehow indicate that this is about multicast dns? If we had some common name for LLMNR + mDNS, it'd be nice to use it here.

Reply:
Renamed to GetMulticastHosts()